agent/alembic/versions/669a641624ed_add_protocol_column_to_pipelines.py

```diff
@@ -0,0 +1,24 @@
+"""add_protocol_column_to_pipelines
+
+Revision ID: 669a641624ed
+Revises: 63aac660c5f0
+Create Date: 2021-01-20 14:55:47.905829
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '669a641624ed'
+down_revision = '63aac660c5f0'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('pipelines', sa.Column('protocol', sa.String))
```

Comment:
I think using a separate column and a property protocol is redundant. `use_schema` property in the config should be enough. And it will be easier to get rid of this when all pipelines will always use protocol 3.0

Reply:
I created a separate column because it's not a part of pipeline configuration, it's rather a flag that shows what pipeline type we should build.
when all pipelines use protocol 3 we'll just drop that column.